res/assets/ui.xml

```diff
@@ -223,4 +224,15 @@
 	<image id="11Legion_Elephants" src="11Legion_Elephants"/>
 	<image id="13Legion_Bulls" src="13Legion_Bulls"/>
 	<image id="14Legion_Centaurs" src="14Legion_Centaurs"/>
+	<!-- Can't seem to get this working. will have to wait for crudelios to fix it -->
+	<!--<image id="LandRouteDot" width="5" height="5">
+		<layer group="181" image="7"/>
+		<layer src="RouteDotMask" mask="alpha"/>
+	</image>
+	<image id="SeaRouteDot" width="5" height="5">
+		<layer group="182" image="7"/>
+		<layer src="RouteDotMask" mask="alpha"/>
+	</image>-->
```

Comment:
What's the issue here?

Reply:
Can’t seem to get the trade dot image to show up with masking. Was hoping you could take a look.